src/Features/Onboarding.php

```diff
@@ -1053,8 +1053,17 @@ public static function reset_task_list() {
 			return;
 		}
 
-		$new_value = 1 === absint( $_GET['reset_task_list'] ) ? 'no' : 'yes'; // phpcs:ignore CSRF ok.
+		$previous_value = 1 === absint( $_GET['reset_task_list'] ) ? 'yes' : 'no'; // phpcs:ignore CSRF ok.
+		$new_value      = 'no' === $previous_value ? 'yes' : 'no'; // phpcs:ignore CSRF ok.
+
 		update_option( 'woocommerce_task_list_hidden', $new_value );
+		wc_admin_record_tracks_event(
```

Comment:
I've deviated from the issue spec here to match the other similar events we track in the Onboarding class. Keen for some feedback here on whether its ok to deviate or whether it should match the spec more exactly.

Reply:
I suppose its best to imagine you're digging through events to determine a percentage of people toggling the task list. I think the first SQL is easier to read:
```
WHERE eventname = 'wcadmin_tasklist_toggled' AND eventprop.status = 'enabled'
```
versus
```
WHERE eventname = 'wcadmin_tasklist_toggled' AND eventprop.new_value = 'yes'
```
Its subjective though, what do you think?

Reply:
Yep I really just focussed on consistency here with existing code, but I think you're right, the former is a much more obvious query. I'll take this approach.